src/client/app/helpers/trusted.snippet.helper.ts

```diff
@@ -0,0 +1,26 @@
+class TrustedSnippet {
+    private _trustedSnippetsKey = 'trusted_snippets';
+
+    isSnippetTrusted(snippetId: string, gistId: string): boolean {
+        let trustedSnippets = JSON.parse(window.localStorage.getItem(this._trustedSnippetsKey));
+        if (!trustedSnippets) {
+            trustedSnippets = {};
+        }
+        /* Samples are automatically trusted. Check local storage for gists. */
+        if (!gistId || trustedSnippets[snippetId]) {
+            return true;
+        }
+        return false;
+    }
+
+    updateTrustedSnippets(snippetId: string): void {
+        let trustedSnippets = JSON.parse(window.localStorage.getItem(this._trustedSnippetsKey));
+        if (!trustedSnippets) {
+            trustedSnippets = {};
+        }
+        trustedSnippets[snippetId] = true;
+        window.localStorage.setItem(this._trustedSnippetsKey, JSON.stringify(trustedSnippets));
+    }
+}
+
+export const trustedSnippet = new TrustedSnippet();
```

Comment:
Can you call this something that tells me what exactly it is such as trustedSnippetManager?